### PR TITLE
Borg module cleanup and fixes

### PR DESCRIPTION
**Pull request by thegrb93**
## About The Pull Request
Makes the robot_model.dm more consistent with TG while also fixing a potential bug introduced by skyrat code.
Adds bloodfilter tool to mediborg which TG borgs have.
Cleans up and organizes the messy jumbled civilian borg modules and removes redundant borgshaker, but adds the vineger and cherry that it had to the specific shakers.

## How This Contributes To The Skyrat Roleplay Experience
Adds some much wanted/needed tools to borg

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
293611735-ad459238-50e8-4b45-826f-035381eee6ba

293612352-22718abd-a04d-417d-9f36-510665a79421

</details>

## Changelog

:cl:
add: Blood filter to medical borg
add: Added cherry jelly and vineger to service borg shakers
del: Remove redundant service borgshaker
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 1]
```diff
@@ -225,6 +225,10 @@
 	new_model.rebuild_modules()
 	cyborg.radio.recalculateChannels()
 	cyborg.set_modularInterface_theme()
+	cyborg.diag_hud_set_health()
+	cyborg.diag_hud_set_status()
+	cyborg.diag_hud_set_borgcell()
+	cyborg.diag_hud_set_aishell()
```
Comment (jjpark-kb) [inline code/modules/mob/living/silicon/robot/robot_model.dm:231]: this should be done modularly.

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 2]
```diff
@@ -266,6 +270,7 @@
 		if(!isnull(details[SKIN_ICON]))
 			cyborg.icon = details[SKIN_ICON]
 			cyborg_icon_override = details[SKIN_ICON] // SKYRAT EDIT ADDITION
+		if(!isnull(details[SKIN_PIXEL_X]))
```
Comment (jjpark-kb) [inline code/modules/mob/living/silicon/robot/robot_model.dm:273]: either needs to be done modularly or marked as a skyrat edit

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 3]
```diff
@@ -805,29 +811,30 @@
...
 		/obj/item/borg/apparatus/beaker/service,
+		/obj/item/borg/apparatus/beaker, // Allows the pickup of different beakers for easier drink mixing
+		// SKYRAT EDIT END
 		/obj/item/reagent_containers/cup/beaker/large, //I know a shaker is more appropiate but this is for ease of identification
-		//Skyrat Edit Start: Borg Buff
-		//obj/item/reagent_containers/condiment/enzyme, //edit - Borg shaker has it
-		/obj/item/borg/apparatus/beaker, // SKYRAT EDIT: allows the pickup of different beakers for easier drink mixing
+		//obj/item/reagent_containers/condiment/enzyme, // SKYRAT EDIT - Borg shaker has it
 		/obj/item/reagent_containers/dropper,
+		/obj/item/reagent_containers/syringe, // SKYRAT EDIT
```
Comment (jjpark-kb) [inline code/modules/mob/living/silicon/robot/robot_model.dm:826]: is there a reason this isn't under that large skyrat edit?

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 4]
```diff
@@ -805,29 +811,30 @@
...
 		/obj/item/rsf,
 		/obj/item/storage/bag/tray,
 		/obj/item/storage/bag/tray, // SKYRAT EDIT: Moves the second tray up to be near the default one
+		/obj/item/cooking/cyborg/power, // SKYRAT EDIT
```
Comment (jjpark-kb) [inline code/modules/mob/living/silicon/robot/robot_model.dm:830]: is there a reason this isn't under the large skyrat edit?

**Comment by jjpark-kb** [comment]
Please complete the reviewed sections-- once that is done, it should look good to go.

File: code/modules/mob/living/silicon/robot/robot_model.dm (hunk 1 above)
Comment (thegrb93) [reply code/modules/mob/living/silicon/robot/robot_model.dm:231]: This is TG code

File: code/modules/mob/living/silicon/robot/robot_model.dm (hunk 2 above)
Comment (thegrb93) [reply code/modules/mob/living/silicon/robot/robot_model.dm:273]: TG code

File: code/modules/mob/living/silicon/robot/robot_model.dm (hunk 3 above)
Comment (thegrb93) [reply code/modules/mob/living/silicon/robot/robot_model.dm:826]: Keeps the modules ordered and organized by their function

File: code/modules/mob/living/silicon/robot/robot_model.dm (hunk 4 above)
Comment (thegrb93) [reply code/modules/mob/living/silicon/robot/robot_model.dm:830]: Keeps the modules ordered and organized by their function

**Comment by thegrb93** [comment]
Replied to comments

**Review by jjpark-kb — APPROVED** [review]

**Pull request merged**